### PR TITLE
Updating coding standard: use of comments

### PR DESCRIPTION
**Pull request by andrsd**

### CODE REVIEW

File: doc/coding_standard.md [hunk 1]
```diff
@@ -130,12 +130,16 @@ call();
 ```
 
 For [doxygen](https://www.doxygen.nl/) style comments `/** */` should be used.
+For single-line doxygen-style comment, use `///`.
```
Comment (ragusa) [inline doc/coding_standard.md:133]: Do you want to mandate `///` or just sugest it as an acceptable alternative for one-liner ?

Comment (ragusa) [reply doc/coding_standard.md:133]: I let you resolve this whichever is best. I will approve right away.

**Review by ragusa — APPROVED** [review]

**Review by wdhawkins — APPROVED** [review]

File: doc/coding_standard.md (hunk 1 above)
Comment (zhardy-lanl) [reply doc/coding_standard.md]: I think that at this point in time we should allow either `/** text */` or `/// text` for single line comments just based on the sheer number of `/** text */` comments that exist in the code. If we move to only `///` we would then need to undertake the massive effort of modifying those existing comments.

File: doc/coding_standard.md [hunk 2]
```diff
@@ -129,13 +129,20 @@ In-code comments should use `//`.
 call();
 ```
 
-For [doxygen](https://www.doxygen.nl/) style comments `/** */` should be used.
+For [doxygen](https://www.doxygen.nl/)-style comments, use `///` for single-line comments
```
Comment (zhardy-lanl) [inline doc/coding_standard.md:132]: If we do not allow `/** */` single line comments, are we going to remove these throughout the codebase?

Comment (andrsd) [reply doc/coding_standard.md:132]: I started working on that. I am almost done from with `framework/`...

Comment (zhardy-lanl) [reply doc/coding_standard.md:132]: Separate PR I take it?

Comment (andrsd) [reply doc/coding_standard.md:132]: Yes

**Review by zhardy-lanl — APPROVED** [review]

**Pull request merged**